### PR TITLE
Add pulp ansible 0.2.0 announcement

### PR DESCRIPTION
**Pull request by melcorr**
Announcing the Ansible 0.2.0 release

### CODE REVIEW

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Pulp Container 2.0.0 is Generally Available
```
Comment (mdellweg) [inline _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]:
```suggestion
title: Pulp Ansible 0.2.0 is Generally Available
```

Comment (melcorr) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: That's some tasty copy pasta right there. 
I had copied over a header.

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# Pulp Ansible 0.2.0 Release Announcement
+
+The Pulp team is pleased to announce the release of the Pulp Ansible 0.2.0 plugin.
+
+## About the Pulp Ansible content plugin
+
+If you’re an Ansible user and do not want to host your private content on Ansible Galaxy, you can add the Pulp Ansible plugin to Pulp. You can then mirror the public Ansible content that you require and use Pulp as an on-premise platform to manage and distribute a scalable blend of public and private Ansible roles and collections across your organization.
+
+With the Pulp Ansible plugin, you can complete the following actions via the Pulp API. Ansible content includes both roles and collections:
```
Comment (mdellweg) [inline _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: "actions" could be a little bit more high level as "workflows"

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+* Mirror a subset of Ansible content on-premise
+* Mirror all of Ansible Galaxy’s content on-premise
+* Store private Ansible content on-premise
+* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
+* Version Ansible content over time and rollback if necessary
+* Support for the new multi-role content type from Galaxy
```
Comment (mdellweg) [inline _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: There is not an action in this. May be it should be "as well as the new multi-role type" in the sentence above.

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+## About the Pulp Ansible content plugin
+
+If you’re an Ansible user and do not want to host your private content on Ansible Galaxy, you can add the Pulp Ansible plugin to Pulp. You can then mirror the public Ansible content that you require and use Pulp as an on-premise platform to manage and distribute a scalable blend of public and private Ansible roles and collections across your organization.
+
+With the Pulp Ansible plugin, you can complete the following actions via the Pulp API. Ansible content includes both roles and collections:
+
+* Mirror a subset of Ansible content on-premise
+* Mirror all of Ansible Galaxy’s content on-premise
+* Store private Ansible content on-premise
+* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
```
Comment (mdellweg) [inline _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]:
```suggestion
* Distribute Ansible content from pulp_ansible to be installable with the ansible-galaxy CLI
```

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+For more information, see the [roles](https://pulp-ansible.readthedocs.io/en/latest/workflows/roles.html) and [collections](https://pulp-ansible.readthedocs.io/en/latest/workflows/collections.html) workflows in the Pulp Ansible documentation.
+
+## Release Features
+
+In this release, which is compatible with Pulp 3.6, the following functionality has been added:
+
+* Users can now associate a remote with a repository and automatically use this remote when syncing the repository. [#7194](https://pulp.plan.io/issues/7194)
+
+As part of this release, artifacts are no longer used when importing collections [#6718](https://pulp.plan.io/issues/6718)
```
Comment (mdellweg) [inline _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: Is this "temporary artifacts are no longer persisted"?
In the end, i sure hope that an artifact is used to represent the collection. ;)

Comment (mdellweg) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: ~~Is the list bullet missing here?~~
Forget that!

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md (hunk 3 above)
Comment (melcorr) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: I will split these into workflows and features maybe?

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md (hunk 4 above)
Comment (melcorr) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: The suggested update is hard for me to understand. 

Would something like this work:

* Distribute Ansible content from `pulp_ansible` and perform actions with the ansible-galaxy CLI

Comment (mdellweg) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: The title to this list states that you can do stuff with the rest api. And there you can create a distribution (that is roughly a URL to serve content).
After you did that, you can use the `ansible-galaxy` command on any consuming machine to install content from this distribution.

Comment (melcorr) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: This is so clear that even I understand. Thank you :D

File: _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md [hunk 6]
```diff
@@ -13,24 +13,27 @@ The Pulp team is pleased to announce the release of the Pulp Ansible 0.2.0 plugi
...
-* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
-* Version Ansible content over time and rollback if necessary
-* Support for the new multi-role content type from Galaxy
+* Create a distribution and use the `ansible-galaxy` command to install content from this distribution
 
 For more information, see the [roles](https://pulp-ansible.readthedocs.io/en/latest/workflows/roles.html) and [collections](https://pulp-ansible.readthedocs.io/en/latest/workflows/collections.html) workflows in the Pulp Ansible documentation.
 
+The Pulp Ansible plugin also has the following features:
+
+* Store private Ansible content on-premise
```
Comment (mdellweg) [inline _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: @bmbouter Do we support this via `ansible-galaxy push`?
I think that would be worth mentioning.

Comment (bmbouter) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: Yes we support uploading using `ansible-galaxy collection publish`. The `ansible-galaxy role --help` doesn't have an upload capability in the client, so we only support the uploading of collections.

Comment (bmbouter) [reply _posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md]: There is no role upload/publish here: https://pulp-ansible.readthedocs.io/en/latest/workflows/roles.html
There are collection publish docs here: https://pulp-ansible.readthedocs.io/en/latest/workflows/collections.html

**Comment by melcorr** [comment]
Thanks @bmbouter  and @mdellweg  
I *think* I have applied all feedback. 
Let me know if there is anything else.

**Pull request merged**